dev/.env.docker-compose

```diff
@@ -11,3 +11,4 @@ DJANGO_DANDI_DANDISETS_EMBARGO_BUCKET_NAME=dandi-embargo-dandisets
 DJANGO_DANDI_WEB_APP_URL=http://localhost:8085
 DJANGO_DANDI_API_URL=http://localhost:8000
 DJANGO_DANDI_JUPYTERHUB_URL=https://hub.dandiarchive.org/
+DJANGO_DANDI_API_VERSION=v0.0.0
```

Comment:
so local containerized instances used by dandi-cli would not have it set, or these compose recipes aren't used for dandi-cli? (attn @jwodder)

Reply:
`.env.docker-compose` is only used when running Docker Compose from within the dandi-archive repository.  The environment variables used by dandi-cli's Docker Compose setup are set in its `docker-compose.yml` file.

Reply:
This variable also defaults to `v0.0.0` if not supplied.